config/manager/BUILD.bazel

```diff
@@ -1,20 +0,0 @@
-# Copyright 2019 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-exports_files(
-    [
-        "manager.yaml",
-    ],
-    visibility = ["//visibility:public"],
-)
```

Comment:
Any reason why this bazel file got removed?

Reply:
As far as I remember it clashed with the entries I added to `config/BUILD.bazel` and removing it fixed it.